docs/content/latest/benchmark/tpcc-ysql/10-wh/tpcc-ysql.md

```diff
@@ -23,14 +51,14 @@ $ ./tpccbenchmark --create=true --load=true --execute=true \
   --nodes=127.0.0.1,127.0.0.2,127.0.0.3
 ```
 
-### TPC-C Benchmark Results
+## 4. TPC-C Benchmark Results
 
 Once the execution is done the TPM-C number along with the efficiency is printed.
 
 ```
-20:57:54,318 (DBWorkload.java:896) INFO  - Rate limited reqs/s: Results(nanoSeconds=1800000300980, measuredRequests=8528) = 4.737776985568823 requests/sec
-20:57:54,318 (DBWorkload.java:901) INFO  - Num New Order transactions : 3799, time seconds: 1800
-20:57:54,318 (DBWorkload.java:902) INFO  - TPM-C: 126
-20:57:54,318 (DBWorkload.java:903) INFO  - Efficiency : 97.97822706065319
-20:57:54,321 (DBWorkload.java:738) INFO  - Output Raw data into file: results/oltpbench.csv
+22:51:04,125 (DBWorkload.java:895) INFO  - Rate limited reqs/s: Results(nanoSeconds=1800000323771, measuredRequests=8536) = 4.742221369225692 requests/sec
+22:51:04,125 (DBWorkload.java:900) INFO  - Num New Order transactions : 3839, time seconds: 1800
+22:51:04,126 (DBWorkload.java:901) INFO  - TPM-C: 127
```

Comment:
Isn't this 127.6? Any reason why we're not showing 1 or 2 digits after decimal point?

Reply:
Will add a review for this and then change this.